react/introduction/how_this_course_will_work.md

```diff
@@ -1,14 +1,14 @@
 
-Make sure you’ve completed our [JavaScript course](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript) before starting this one. We can’t emphasize enough how imperative it is to have a fair understanding of JavaScript before diving into React. At the end of the day, React *is* vanilla JavaScript, so being able to navigate in JavaScript land is paramount to building a successful React project.
+Make sure you’ve completed our [JavaScript course](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript) before starting this one. We can’t emphasize enough how important it is to have a solid understanding of JavaScript before diving into React. At the end of the day, React *is* vanilla JavaScript, so being able to navigate JavaScript is paramount to building a successful React project.
 
 ### The journey ahead
 
-All the lessons follow a similar structure that you're accustomed to from your prior courses. There'll be lesson content followed by assignments. We've also incorporated interactive examples via CodeSandbox to demonstrate the concepts learned in those lessons.
+All the lessons follow a similar structure that you'll be accustomed to from prior courses: lesson content followed by assignments. We've also incorporated interactive examples via CodeSandbox to demonstrate the concepts you'll learn in those lessons.
 
-As always, we're going to preface this by saying that learning a new technology can be a frustrating endeavor, but we're confident you can persevere. There might be times when you think to yourself, “Hey, I can do this using vanilla JavaScript alright, why am I even learning React”, and that's fine. As you work through the lessons, you'll soon realize how convenient React can be in front-end development. Remember the dreaded [Todo List project](https://www.theodinproject.com/lessons/node-path-javascript-todo-list) from the JavaScript course? It’ll take you a few minutes in React to get the core functionality down. That will surely get you excited. By the end of the course, you’ll be a React Guru!
+As always, we'll preface this by saying that learning a new technology can be a frustrating endeavor, but we're confident you can persevere. There might be times when you think to yourself, “I can do this with vanilla JavaScript, why am I even learning React?” and that's fine. As you work through the lessons, you'll soon realize how convenient React can be in front-end development. Remember the dreaded [Todo List project](https://www.theodinproject.com/lessons/node-path-javascript-todo-list) from the JavaScript course? In React, it would take just a few minutes to build the core functionality. That should get you excited. By the end of the course, you’ll be a React Guru!
```

Comment:
Just to avoid specific time lengths, as we don't want it to cause users to worry if it doesn't take them only a few minutes or anything:

```suggestion
As always, we'll preface this by saying that learning a new technology can be a frustrating endeavor, but we're confident you can persevere. There might be times when you think to yourself, “I can do this with vanilla JavaScript, why am I even learning React?” and that's fine. As you work through the lessons, you'll soon realize how convenient React can be in front-end development. Remember the dreaded [Todo List project](https://www.theodinproject.com/lessons/node-path-javascript-todo-list) from the JavaScript course? In React, it would typically take less time to build the core functionality. That should get you excited. By the end of the course, you’ll be a React Guru!
```